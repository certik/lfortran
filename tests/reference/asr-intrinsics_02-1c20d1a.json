{
    "basename": "asr-intrinsics_02-1c20d1a",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_02.f90",
    "infile_hash": "62898f6506eceb32aa540718b4d9de7cd0f9942d9cf5eb181748eba9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_02-1c20d1a.stdout",
    "stdout_hash": "af0eb6729e8dd24a57e15dc26f54e7da4c7091a3d659f074f9ea2a88",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}