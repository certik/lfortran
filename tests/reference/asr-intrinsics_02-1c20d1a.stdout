(TranslationUnit (SymbolTable 1 {intrinsics_02: (Program (SymbolTable 2 {sin: (ExternalSymbol 2 sin 4 sin lfortran_intrinsic_math [] sin Private), sin@ssin: (ExternalSymbol 2 sin@ssin 4 ssin lfortran_intrinsic_math [] ssin Private), x: (Variable 2 x Local () () Default (Real 4 []) Source Public Required .false.)}) intrinsics_02 [] [(= (Var 2 x) (FunctionCall 2 sin@ssin 2 sin [((RealConstant   1.50000000000000000e+00 (Real 4 [])))] (Real 4 []) (RealConstant   9.97494986604054446e-01 (Real 4 [])) ()) ()) (Print () [(Var 2 x)] () ())]), iso_c_binding: (IntrinsicModule lfortran_intrinsic_iso_c_binding), iso_fortran_env: (IntrinsicModule lfortran_intrinsic_iso_fortran_env), lfortran_intrinsic_builtin: (IntrinsicModule lfortran_intrinsic_builtin), lfortran_intrinsic_math: (IntrinsicModule lfortran_intrinsic_math)}) [])
